app/middlewares/errors.js

```diff
@@ -6,7 +6,8 @@ const DEFAULT_STATUS_CODE = 500;
 const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
   [errors.DEFAULT_ERROR]: 500,
-  [errors.BAD_REQUEST_ERROR]: 400
+  [errors.BAD_REQUEST_ERROR]: 400,
+  [errors.UNAUTHORIZED_ERROR]: 401
```

Comment:
Unauthorized use to be 403 (aka forbidden). 401 use to be authentication error.

Reply:
I have created a forbidden error with the code 403. 👀